inventory_management_system_api/auth/jwt_bearer.py

```diff
@@ -43,6 +43,8 @@ async def __call__(self, request: Request) -> str:
         if not self._is_jwt_access_token_valid(credentials.credentials):
             raise HTTPException(status_code=status.HTTP_403_FORBIDDEN, detail="Invalid token or expired token")
 
+        request.state.token = credentials.credentials
```

Comment:
Not sure if this is a nice thing to do as there are not many examples when I searched for it on GitHub. The alternative would be to do `request.headers.get("Authorization").split("Bearer ")[1]` in the endpoints.